dealership/vehicles.py

```diff
@@ -1,15 +1,35 @@
 class Vehicle(object):
+    SALES_MULTIPLIER = None
+    PURCHASE_MILEAGE_MULITPLIER = None
+    INTEREST_RATE = None
+    LEASE_MULTIPLIER = None
     def __init__(self, maker, model, year, base_price, miles):
-        pass
-
+        self.maker = maker
+        self.model = model
+        self.base_price = base_price
+        self.miles = miles
+        self.year = year
+        
+    def sale_price(self):
+        return self.SALES_MULTIPLIER * self.base_price
+    def purchase_price(self):
+        return self.sale_price() - self.PURCHASE_MILEAGE_MULITPLIER * self.miles
+    
 
 class Car(Vehicle):
-    pass
-
-
+    SALES_MULTIPLIER = 1.2
```

Comment:
I think this is a great solution - for vehicle subclasses you have defined its multipliers, I haven't thought about that in my own code :)